bin/resources/GameIndex.yaml

```diff
@@ -31211,6 +31241,12 @@ SLPM-68516:
 SLPM-69001:
   name: "PictureParadise Club"
   region: "NTSC-J"
+SLPM-69002:
+  name: "NetFront"
+  region: "NTSC-J"
+SLPM-69004:
+  name: "EGBrowser Light for I-O Data Device, Inc."
```

Comment:
is the comma intentional? seems out of place

Reply:
Yes : https://buyee.jp/item/yahoo/auction/c1051324223

Reply:
I can't read Japanese, is it in one of the pictures, or something?

If I google it, I don't see that comma anywhere.

Reply:
ah yeah, I couldn't see that, thanks